.github/workflows/docker.yml

```diff
@@ -0,0 +1,33 @@
+name: docker
+
+on:
+  push:
+    branches:
+      - "master"
```

Comment:
Do we want to rename our master branch to main? It's quite easy nowadays (see https://github.com/github/renaming). This isn't blocking this PR but this reference reminded me that we should probably do this.

Reply:
Yeah we should. Does it "rebase" / update nicely for people with existing checkouts etc? If so,. I can do now.

Reply:
No unfortunately:

> Also, GitHub does not perform any redirects if users perform a git pull for the previous branch name

https://docs.github.com/en/github/administering-a-repository/renaming-a-branch#about-renaming-branches